src/command_modules/azure-cli-resource/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.0.30
+++++++
+* Add management-group commands
```

Comment:
This needs to be more explicit especially since these commands live in the command tree that is predominantly in a different module. 
"* add `account management-group` commands."

Reply:
Ok will make the change. Also, just checked the travis build. Will make the changes.